### ISSUE TITLE
색깔 접근성 문제 해결 구현

### ISSUE THREAD

**Issue by DaleSeo** [DaleStudy/leaderboard#170]
#163 구현

**Comment by DaleSeo** [comment]
조치가 끝나면 https://github.com/DaleStudy/leaderboard/pull/162#discussion_r1898096258 에서 잠시 꺼둔 접근성 테스트를 활성화시켜주시기 바랍니다.

**Comment by DaleSeo** [comment]
@SamTheKorean 님, 접근성 테스트 활성화하는 부분은 별도 PR에서 진행하실 계획이신가요? 이슈가 완료 처리되어 있어서 여쭙니다!

**Comment by SamTheKorean** [comment]
@DaleSeo 네 달레님! 설정을 gui로 끄는 것이라고 착각하여 pr 병합했다가 병합후에 코드에서 설정을 지워주면 되는 걸 알아서 지금 진행중에 있습니다! 별도 pr만들고 해당 이슈와 묶어도 될까요?

**Comment by SamTheKorean** [comment]
아직 해결해야할 문제가 있어서 타 pr 열고 해당 task는 progress로 옮겼습니다. @DaleSeo
